src/styles/_catalog.scss

```diff
@@ -14,6 +16,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform 300ms;
+
+    &:hover {
+      transform: translate3d(0, 0, 20px);
```

Comment:
You should smoothly (during 300ms) increase the card by 20 percent on hover (neighboring cards should not be affected)

Now in the hover your cards do not have a smooth increase and overlap each other

Use for it `transform: scale`